### ISSUE TITLE
Create action that can change the audio output

### ISSUE THREAD

**Issue by dustinrue** [dustinrue/ControlPlane#167]
Seems more useful now that you might want to key it off of an AirPlay device being available.  Should allow user to output to a given AirPlay device.

**Comment by l00mi** [comment]
+1. Different context have more and more different speakers ...

**Comment by jeremymeyers** [comment]
+1 long after the fact

**Comment by ChristinWhite** [comment]
@jeremymeyers The last commit was four years ago, this seems to be entirely unmaintained and no one else has taken over. I'm surprised it's still working with changes in the last two OS releases.

**Comment by jeremymeyers** [comment]
@ChristinWhite spoke with the author on twitter about it. but yeah sad

**Comment by ChristinWhite** [comment]
Yeah, it's unfortunate when a passion project has to get put on indefinite hold and no one can take over, wish I had the time and skill to contribute.

**Comment by dustinrue** [comment]
The original codebase never shipped this. Tracked for the rewrite as #571, which includes a full implementation spec using CoreAudio's public APIs.
